### PR TITLE
acpica: add license

### PR DESCRIPTION
**Pull request by chenrui333**
- acpica: add license

### CODE REVIEW

File: Formula/acpica.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Acpica < Formula
   homepage "https://www.acpica.org/"
   url "https://acpica.org/sites/acpica/files/acpica-unix-20200717.tar.gz"
   sha256 "cb99903ef240732f395af40c23b9b19c7899033f48840743544eebb6da72a828"
+  license ["Intel-ACPI", "Intel", "GPL-2.0-only"]
```
Comment (Rylan12) [inline Formula/acpica.rb:6]: According to the SPDX license page for [`Intel`](https://spdx.org/licenses/Intel.html#licenseText), it has been deprecated:

> This license has been deprecated. A note at the top of the OSI license page states, "The Intel Open Source License for CDSA/CSSM Implementation (BSD License with Export Notice) (Intel has ceased to use or recommend this license)"

However, it's not marked as a deprecated identifier by SPDX. I think that means it's okay but I'm not positive.

---

I see some contradicting facts on the website. [This license](https://acpica.org/sites/acpica/files/licensing.txt) indicates what you've put here: `Intel-ACPI`, `Intel`, or `GPL-2.0-only`.

[This page](https://www.acpica.org/licensing), however, indicates that it depends on which package you've downloaded. It says that `Intel` is for `acpica-unix` (which is what this formula uses) and `GPL-2.0` or `BSD-2-Clause` for certain other packages. However, the [downloads page](https://acpica.org/downloads) says that the dual-license (GPL/BSD) is in the `acpica-unix` version and that the other packages are deprecated.

However, the [license header on the source files](https://acpica.org/sites/acpica/files/licensing.txt) seems to be what we should go with. It definitely indicates `Intel-ACPI` and `GPL-2.0-only`, but I think the third is `BSD-3-Clause`. They're very similar but the `Intel` license explicitly mentions the Intel Corporation:

> 3. Neither the name of the **Intel Corporation** nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.

The license provided only says:

> 3. Neither the names of the above-listed copyright holders nor the names of any contributors may be used to endorse or promote products derived from this software without specific prior written permission.

Comment (chenrui333) [reply Formula/acpica.rb:6]: yeah, `Intel` license is very much the same as `BSD-3-Clause`, but with Intel corporation as holder.

Comment (chenrui333) [reply Formula/acpica.rb:6]: The license ref I was using is from the source code header. I should put in the PR body. With that being said, `Intel` as equivalent to `BSD-3-Clause` is what I used in here.

**Review by Rylan12 — APPROVED** [review]

**Comment by chenrui333** [comment]
mark as `do not merge` to have more eyes on this.

**Comment by chenrui333** [comment]
cc @jonchang

**Comment by jonchang** [comment]
LGTM

**Pull request merged**